app/services/decisions/capital_decision_tree.rb

```diff
@@ -62,26 +68,23 @@ def after_properties
         property_owners.create! if incomplete_property_owners.blank?
         return edit(:property_owners, property_id: property)
       end
-      # TODO: Route to appropriate property page loop once built
-      edit(:saving_type) # Placeholder to join up flow
+      edit(:properties_summary) # Placeholder to join up flow
     end
 
     def after_property_address
       if form_object.has_other_owners.to_s == YesNoAnswer::YES.to_s
         property_owners.create! if incomplete_property_owners.blank?
         return edit(:property_owners, property_id: property)
       end
-
-      # TODO: Route to appropriate property page loop once built
-      edit(:saving_type) # Placeholder to join up flow
+      edit(:properties_summary) # Placeholder to join up flow
     end
 
     def incomplete_property_owners
       property_owners.reject(&:complete?)
     end
 
     def after_property_owner
-      edit(:saving_type) # Placeholder to join up flow
+      edit(:properties_summary) # Placeholder to join up flow
```

Comment:
We can actually just put this in the tree directly if there's no conditional flow logic.
```
 def destination
      case step_name...
      ...
      when :property_owners
             edit(:properties_summary)
             ```